### PR TITLE
Updated params and example code for all endpoints in the Actions section

### PR DESCRIPTION
**Pull request by EE-GSlomin**
Relocated contract recording endpoints to _api_contractrecording.md
Moved All Cameras On/Off to _api_cameras.md

### CODE REVIEW

File: source/includes/_api_action.md [hunk 1]
```diff
@@ -1,113 +0,0 @@
-# Action
```
Comment (georgekarrv) [inline source/includes/_api_action.md]: With this file removed build fails because it can't locate it in index, gotta remove the ref to it there.

Comment (georgekarrv) [inline source/includes/_api_action.md]: with that change everything looks good

File: source/includes/_api_camera.md [hunk 2]
```diff
@@ -1356,3 +1356,55 @@ HTTP Status Code    | Data Type
...
+
+> Response 
+
+Used to turn on all cameras in the caller user’s account. Requires the account_superuser permission.
+
+### HTTP Request
+
+`POST https://login.eagleeyenetworks.com/g/action/allon`
+
+### Error Status Codes <!--TODO: Investigate archiver error codes -->
```
Comment (jtallieu) [inline source/includes/_api_camera.md]: Comments need to be on their own line else they show up in the output.
image

File: source/includes/_api_contractrecording.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+Update a Recording
+
+### HTTP Request
+
+`POST https://login.eagleeyenetworks.com/g/recording`
+
+Parameter       	| Data Type   	| Description
+---------       	| ----------- 	| -----------
+**recording_key**   | string      	| Unique identifier (within an account) of a recording
+meta 				| object 		| Meta data. This is meant to be a generic object that can store any data that is needed, so the properties are not predefined.
```
Comment (devinrosen) [inline source/includes/_api_contractrecording.md]: Does the meta object have to be JSON?  If so we should specify.  Just making assumptions (have never used recording API)

Comment (EE-GSlomin) [inline source/includes/_api_contractrecording.md]: good question, i'll dig into it.

**Pull request closed without merging**